packages/javascript-sdk/project.json

```diff
@@ -81,7 +81,7 @@
           "commitMessageFormat": "release: ${version} [skip ci]",
           "remote": "origin",
           "push": true,
-          "postTargets": ["javascript-sdk:deploy:beta"],
+          "postTargets": ["javascript-sdk:deploy:beta", "javascript-sdk:github:beta"],
```

Comment:
Are you able to explain what we're adding here, with `javascript-sdk:github:beta` and `javascript-sdk:github`? It's not obvious to me and would like to know more. Thanks.

Reply:
its a post target, so after this runs, it will run that nx target.